pkg/config/transport.go

```diff
@@ -124,5 +174,35 @@ func serverFinalizer(logger log.Logger) kithttp.ServerFinalizerFunc {
 				"statusCode": code,
 			},
 		)
+
+		requestLatency := kitprom.NewHistogramFrom(
```

Comment:
Did you try to execute more than one request against this implementation? I suspect it will cause a panic. 👹